lib/jemoji.rb

```diff
@@ -3,9 +3,12 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Emoji < Jekyll::Generator
+  class Emoji < Jekyll::Converter
     attr_reader :config
     safe true
+    priority :low
+
+    EMOJIABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
 
```

Comment:
I'm trying to think of the scenario where GitHub doesn't update a gem every time someone devises a new markup extension to add here. Can't `matches` just always result in `true`?